src/smart-components/role/add-role/inventory-groups-role.js

```diff
@@ -133,78 +148,87 @@ const InventoryGroupsRole = (props) => {
     formOptions.change('inventory-group-permissions', groupsPermissionsDefinition);
   }, [state]);
 
-  const makeRow = (permissionId) => {
-    const options = Object.values(resourceTypes?.[permissionId] ?? {});
+  const makeRow = (permission, index) => {
+    const options = Object.values(resourceTypes?.[permission] ?? {});
+
     return (
-      <React.Fragment key={`${permissionId}`}>
-        <GridItem md={4} sm={12}>
-          <FormGroup label={permissionId} isRequired />
-        </GridItem>
-        <GridItem md={8} sm={12}>
-          <Tooltip content={<div>{intl.formatMessage(messages.inventoryGroupsTooltip)}</div>}>
-            <Select
-              className="rbac-m-resource-type-select"
-              variant={SelectVariant.checkbox}
-              typeAheadAriaLabel={intl.formatMessage(messages.inventoryGroupsTypeAheadLabel)}
-              aria-labelledby={permissionId}
-              selections={state[permissionId].selected.map(({ name }) => name)}
-              placeholderText={intl.formatMessage(messages.selectGroups)}
-              onSelect={(event, selection) =>
-                onSelect(event, selection, selection === intl.formatMessage(messages.selectAll, { length: options?.length ?? 0 }), permissionId)
-              }
-              onToggle={(isOpen) => {
-                // TODO: persist filter state when https://github.com/patternfly/patternfly-react/issues/9490 is resolved
-                !isOpen && state[permissionId].filterValue?.length > 0 && fetchData([permissionId]);
-                dispatchLocally({ type: 'toggle', key: permissionId, filterValue: '', page: 1, isOpen });
-              }}
-              onClear={() => clearSelection(permissionId)}
-              onFilter={(event) => {
-                if (event) {
-                  dispatchLocally({ type: 'setFilter', key: permissionId, filterValue: event.target.value });
-                  debouncedFetch(() => fetchData([permissionId], { name: state[permissionId].filterValue }), 2000);
+      <React.Fragment key={`${permission}`}>
+        <Grid hasGutter>
```

Comment:
```suggestion
        <Grid>
```
I don't think it's necessary. It consumes a lot of space on small devices.